hardware/ITGWBase.cpp

```diff
@@ -15,7 +15,7 @@
 #include "itgw/ITGWSwitchIntertechno.h"
 #include "itgw/ITGWSwitchBrennenstuhl.h"
 
-
+#define _DEBUG
```

Comment:
That is not the way...
Please... install VS2017 and just debug ? It's not that complicated... and windows is not a to bad OS either...